docs/investigation/composed-action/REAMDE.md

```diff
@@ -0,0 +1,163 @@
+## Composed Action
+
+The composed Action refers to a syntax sugar where you can tell Capact Engine to run a given set of Action and monitor them till all are finished.
+
+To run a single Action you need to:
+1. Create Action:
+    ```
+    capact action create --name psql-install cap.interface.database.postgresql.install --parameters-from-file /tmp/psql-install.yaml
+    ```
+
+4. Wait unit it's rendered:
+    ```
+    capact action get psql-install
+    ```
+
+7. Run it:
+    ```
+    capact action run psql-install
+    ```
+
+10. Wait until it's finished:
+    ```
+    capact action watch psql-install
+    ```
+
+13. Extract the output TypeInstances:
+    ```
+    capact action get psql-install -ojson | jq -r '.Actions[].output.typeInstances | map(select(.typeRef.path == "cap.type.database.postgresql.config"))'
+    ```
+
+
+And if you want to use the created PostgreSQL database for other Action, you need to repeat all those steps manually. For now, we overcome this issue by creating Implementation "umbrella workflows". This, unfortunately, results in an additional boilerplate and cause that steps are not atomic causing problem such as https://github.com/capactio/capact/issues/695 or https://github.com/capactio/capact/issues/563. Additionally, such approach make the individual Action less reusable as they don't have a single responsibility.
+
+### ComposedAction
+
+The main responsibility of the Composed Action CR is to schedule a set of Action, watch their execution and pass the input parameters between the steps.
+
+```yaml
+apiVersion: core.capact.io/v1alpha1
+kind: ComposedAction
+metadata:
+  name: "full-spec"
+spec:
+  steps:
+    psql:
+      interface:
+        path: "cap.interface.database.postgresql.install"
+      input:
+        input-parameters:
+          raw:
+            data: |
+              superuser:
+                username: superuser
+              defaultDBName: postgres
+    create-user:
+      depends: ["psql"] # this can be removed, we can create own DAG based on the inputs.
```

Comment:
Maybe we can rename this feld to `dependencies`, `depends-on` or `dependsOn` or `needs` (like GH Actions). It will be more readable 🙂 

```suggestion
      depends: ["psql"] # this can be removed, we can create own DAG based on the inputs.
```

Reply:
The `depends` was taken from the `argo` syntax. Personally, I don't have a strong opinion about the property name. I added all alternatives to the doc 👍